src/App.jsx

```diff
@@ -1,7 +1,9 @@
 import React, { Component } from 'react'
-import { Button, Col, Grid, Nav, Navbar, NavbarBrand, NavItem, Row } from 'react-bootstrap'
+import { Alert, Button, Col, Grid, Nav, Navbar, NavbarBrand, NavItem, Row } from 'react-bootstrap'
 import { Link } from 'react-scroll'
 import { Picture } from 'react-responsive-picture'
+import moment from 'moment'
```

Comment:
Thoughts on something simpler like `(new Date).toLocaleDateString()` rather than pulling in a new library? Not necessarily as guaranteed on consistency (more of this depends on the user's OS settings), but I think that might be okay?